Documentation/design-docs/hw-intrinsics.md

```diff
@@ -12,47 +12,89 @@ There is a design document for the Arm64 intrinsics: https://github.com/dotnet/c
 
 The reference assemblies for the hardware intrinsics live in corefx, but all of the implementation is in the coreclr repo:
 
-* The C# implementation lives in coreclr/src/shared/System/Runtime/Intrinsics. These are little more than skeleton methods that usually invoke themselves recursively (see Indirect invocation support below).
+* The C# implementation lives in coreclr/src/System.Private.CoreLib/shared/System/Runtime/Intrinsics. These are little more than skeleton methods that are only compiled if needed for indirect invocation.
+
+  * Note that they also exist in the dotnet/corefx repo in src\Common\src\CoreLib\System\Runtime\Intrinsic. These are mirrored between the two repositories.
```

Comment:
nit: These are mirrored to a quite a bit more than just corefx/coreclr. It is also mirrored to corert, mono, and possibly others.